File: go-client-sdk/pkg/models/shared/codatclientsapiclientcontractcompanysettings.go
package shared

type CodatClientsAPIClientContractCompanySettings struct {
	CompanyID               *string `json:"companyId,omitempty"`
	OfflineConnectorInstall *bool   `json:"offlineConnectorInstall,omitempty"`
	OneTimeSync             *string `json:"oneTimeSync,omitempty"`
}

File: typescript-client-sdk/src/sdk/models/shared/codatclientsapiclientcontractcompanysettings.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";
import { Expose } from "class-transformer";


export class CodatClientsApiClientContractCompanySettings extends SpeakeasyBase {
  @SpeakeasyMetadata()
  @Expose({ name: "companyId" })
  companyId?: string;

  @SpeakeasyMetadata()
  @Expose({ name: "offlineConnectorInstall" })
  offlineConnectorInstall?: boolean;

  @SpeakeasyMetadata()
  @Expose({ name: "oneTimeSync" })
  oneTimeSync?: string;
}